README.md

```diff
@@ -375,12 +375,28 @@ principles when deciding what `{X}.{Y}.{Z}` should be. See https://github.com/op
 for an example.
 
     ```shell
+    # Make code changes and create tag
     git tag image-v1.2.3
-    git push origin --tags
-    # Create PR here
-    # Typically only team leads can push tags to upstream, so they will need to continue by
-    # checking out your fork and then running
-    git push upstream --tags
+
+    # Push tag to origin
+    git push origin image-v1.2.3
```

Comment:
Does this need `--tags`?

Reply:
No, `--tags` is not needed if you specify the tag name.

Reply:
Cool, sorry for the noise.